support-models/src/main/scala/com/gu/support/catalog/Product.scala

```diff
@@ -41,28 +41,35 @@ case object SupporterPlus extends Product {
   private def productRatePlan(
       id: String,
       billingPeriod: BillingPeriod,
+      versionOptions: SupporterPlusVersionOptions,
   ) =
     ProductRatePlan(
       id,
       billingPeriod,
       NoFulfilmentOptions,
-      NoProductOptions,
+      versionOptions,
```

Comment:
I am using the `ProductOptions` concept to model two version of the same product. This is used by newspaper to differentiate different flavours of sub so works well here.